Add code of conduct

lukpueh opened the pull request:
~blocks on / includes commit from https://github.com/Lind-Project/lind-wasm-docs/pull/37~ (merged)

Copies Contributor Covenant Code of Conduct v2.1 and add to Contributing section.

lukpueh commented on docs/contribute/conduct.md:
```diff
@@ -0,0 +1,133 @@
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
TODO: INSERT CONTACT METHOD

lukpueh replied:
cc @JustinCappos, @rennergade, @Yaxuan-w

Yaxuan-w replied:
Thanks for looping me in! I missed the notification for this PR earlier, so thanks for re-pinging too! I'm happy to serve as a contact person if the group thinks it's appropriate. Would it make sense to also set up a shared email address so that reports can be handled more systematically? (I recalled we've discussed this ~6 months ago, would like to hear more thoughts from @JustinCappos and @rennergade )

JustinCappos replied:
@marcelamelara , I would propose that we list a few email addresses of people they can report to.   Perhaps you and I can be potential contacts?

marcelamelara replied:
I actually like the idea of having a dedicated email address for this, rather than listing individual email addresses. This email address can then forward to actual contacts. Then we don't need to worry as much about keeping the contacts up-to-date in this doc.

JustinCappos replied:
Okay, we can do this.   We should also encourage people to reach out to us individually, if they prefer.   Some incidents are such they won't want to use an impersonal email address.

marcelamelara replied:
That's fair. We will presumably have a "Team" page with our contact info at some point, and we can link to that from here as well.

lukpueh replied:
We have a ["Team" page](https://lind-project.github.io/lind-wasm-docs/community/team/), but it does not include contact infos. I can create an issue for that.

In the meanwhile, I'll add the dedicated reporting email address here.

lukpueh commented:
This is ready for another round of review!

I added a dedicated address, and also moved the code of conduct to the community section (to make the contributing section less convoluted).

We can point to individual team member addresses in https://github.com/Lind-Project/lind-wasm/issues/188.

JustinCappos reviewed (approved):
Looks good now.
